enterprise/web.core/licenseinfo.xml

```diff
@@ -100,4 +100,10 @@
         <license ref="CDDL-1.1" />
         <comment type="CATEGORY_B" />
     </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/web/taglib/resources/web-jsptaglibrary_3_0.xsd</file>
+        <file>src/org/netbeans/modules/web/taglib/resources/web-jsptaglibrary_3_0.mdd</file>
```

Comment:
The mdd has a `EPL-2.0` or `GPLv2.0-CP` header

Reply:
I created the mdd file and copy the header from the other mdd files and did not think about the license, Can I change the header to EPL or should I left it as it is?

Reply:
Please have a look at the comment from `enterprise/j2ee.dd/licenseinfo.xml` I think that should cover it, if not feel free to indicate and I'll retry :smile: